sagemathinc/cocalc
# Fix miss-attribution on marking read

# Description
Correctly use table set method to correctly merge changes to entries.

Also fixes timing error when checking many mentions as seen at the same time (the table would re-report old data for a brief time).

# Testing Steps
1. Mention an account
1. Have that account mark it as read
1. That mention should still be attributed to the correct sender

Many mentions:
1. Create many mentions
1. Check all of them as read quickly (should be able to just click one one and the next will fall into the same space)
1. There shouldn't be any render jank

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
